dev/kyuubi-tpcds/src/main/scala/org/apache/kyuubi/tpcds/DataGenerator.scala

```diff
@@ -51,7 +51,9 @@ object DataGenerator {
 
   import spark.implicits._
 
-  val catalog_sales: TableGenerator = TableGenerator("catalog_sales",
+  // format: off
```

Comment:
what's `// format: off` mean?

Reply:
scalafmt will not format codes between `// format: off` and `// format: on`